indico/modules/rb_new/controllers/backend/bookings.py

```diff
@@ -143,3 +145,16 @@ class RHRoomSuggestions(RHRoomBookingBase):
     @use_args(search_room_args)
     def _process(self, args):
         return jsonify(get_suggestions(args, limit=NUM_SUGGESTIONS))
+
+
+class RHBookingDetails(RHRoomBookingBase):
+    def _process_args(self):
+        self.booking = Reservation.get_one(request.view_args['booking_id'])
+
+    def _process(self):
+        attributes = reservation_details_schema.dump(self.booking).data
```

Comment:
why `attributes`? Isn't it more like `booking_details`?

Reply:
well, it is, but the whole thing is booking details, if you look in the redux state, then you have this data under bookings/details/ so I thought it may be strange to use details again